plugins/linguist-backend/src/api/index.ts

```diff
@@ -14,4 +14,5 @@
  * limitations under the License.
  */
 
-export { LinguistBackendApi } from './LinguistBackendApi';
+export { LinguistBackendClient } from './LinguistBackendClient';
```

Comment:
You should be able to remove this export right? I'm thinking this is the default client that you get when you don't supply your own :)

Reply:
Yes, good call. I've removed this 👍